python/lsst/ts/atspectrograph/atspec_csc.py

```diff
@@ -64,29 +64,29 @@ def __init__(self, config_dir=None, initial_state=salobj.State.STANDBY,
 
         self.timeout = 5.
 
-    async def end_start(self, id_data):
+    async def end_start(self, data):
         """end do_start; called after state changes.
 
         This method call setup on the model, passing the selected setting.
 
         Parameters
         ----------
-        id_data : `CommandIdData`
-            Command ID and data
+        data : `ATSpectrograph_command_start`
```

Comment:
I suggest you remove the back ticks because this cannot be turned into a link, so it is almost certain to cause sphinx errors. Similarly for the other data arguments.